pkgs/development/python-modules/pyvips/default.nix

```
@@ -0,0 +1,38 @@
+{ buildPythonPackage, fetchPypi, python, pytestrunner, gcc, glib, cffi
+, python3Packages, pkgconfig, stdenv, vips, lib}:
```

Remove unused and invalid packages
```suggestion
{ buildPythonPackage, fetchPypi, pytestrunner, glib, cffi
, pkg-config, vips, lib}:
```